src/autofill-utils.js

```diff
@@ -92,6 +93,7 @@ const setValueForSelect = (el, val) => {
             // Events fire on the select el, not option
             events.forEach((ev) => el.dispatchEvent(ev))
             option.selected = true
+            events.forEach((ev) => el.dispatchEvent(ev))
```

Comment:
Some forms do stuff after the field is filled (like extra formatting). By firing the events again we make sure everything works as expected.